### PR TITLE
Relicense the project to LGPL-2.1-or-later

### LINKED ISSUE #462: Change the project license to LGPL-2.1-or-later
Hi all,
We would like to contribute BlueChi project to Eclipse Foundation and as a part of the contribution process we were encouraged to change project license to LGPL-2.1-or-later to ease the contribution process.
In order to change project license we need a consent from all project contributors:

- [x] Ygal Blum
- [x] Sandro Bonazzola
- [x] Pierre-Yves Chibon
- [x] Ed Chong
- [x] Artiom Divak
- [x] Steve Dunnagan
- [x] Michael Engel
- [x] Veronika Fuxova
- [x] Mark Kemel
- [x] Minsoo Kim
- [x] Douglas Schilling Landgraf
- [x] Alexander Larsson
- [x] Lokesh Mandvekar
- [x] Martin Perina
- [x] Yariv Rachmani
- [x] Erik Sjölund
- [x] Petr Šplíchal
- [x] Paul Wallrabe
- [x] Daniel Walsh

Please add your agreement or disagreement with the license change by placing a comment to this issue.

### PR DESCRIPTION
**Pull request by mwperina**
As a part of contributing the project to Eclipse Foundation we need to
relicense the project from GPL-2.0-or-later to LGPL-2.1-or-later.

Fixes: https://github.com/containers/bluechi/issues/462
Signed-off-by: Martin Perina <mperina@redhat.com>

### CODE REVIEW

**Comment by mwperina** [comment]
**IMPORTANT**:

- DO NOT MERGE UNTIL https://github.com/containers/bluechi/issues/462 IS FULLY APPROVED
- BEFORE MERGING REBASE MANUALLY AND RUN FOLLOWING COMMANDS TO MAKE SURE NO FILE CONTAINS INVALID LICENSE
    ```shell
    ./build-scripts/spdx-header-check.sh
    git grep GPL | grep -v 'LGPL-2.1'
    git grep 'SPDX-License-Identifier' | grep -v 'LGPL-2.1' | grep -v 'CC0-1.0'
    ```

**Comment by rhatdan** [comment]
LGTM

**Comment by dougsland** [comment]
LGTM

**Comment by rhatdan** [comment]
Please remove DRAFT flag.

**Comment by engelmi** [comment]
> **IMPORTANT**:
> 
>     * DO NOT MERGE UNTIL [Change the project license to LGPL-2.1-or-later #462](https://github.com/containers/bluechi/issues/462) IS FULLY APPROVED
> 
>     * BEFORE MERGING REBASE MANUALLY AND RUN FOLLOWING COMMANDS TO MAKE SURE NO FILE CONTAINS INVALID LICENSE
>       ```shell
>       ./build-scripts/spdx-header-check.sh
>       git grep GPL | grep -v 'LGPL-2.1'
>       git grep 'SPDX-License-Identifier' | grep -v 'LGPL-2.1' | grep -v 'CC0-1.0'
>       ```

We now have all necessary approvals in #462. 
I've also run the commands listed and checked manually again to make sure no file is missing. 
Removing the draft status.

**Review by ygalblum — APPROVED** [review]
LGTM

**Pull request merged**
